src/main/g8/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=0.13.17
```

Comment:
We should make that sbt1 at some point, but not needed right now, added issue #42